<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SectionControlWidget</class>
 <widget class="QWidget" name="SectionControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="sectionLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="sectionSettingsGroupBox">
     <property name="title">
      <string>Section Settings</string>
     </property>
     <layout class="QGridLayout" name="sectionSettingsLayout">
      <item row="1" column="1">
       <widget class="QSpinBox" name="layerSpinBox">
        <property name="toolTip">
         <string>Sets the number of Layers belonging to this Section</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="activeSectionLabel">
        <property name="text">
         <string>Active Section</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="activeSectionComboBox">
        <property name="toolTip">
         <string>The currently selected Section</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="layerLabel">
        <property name="text">
         <string>Layers</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="layerGroupBox">
     <property name="title">
      <string>Layer Settings</string>
     </property>
     <layout class="QGridLayout" name="layerLayout">
      <item row="1" column="1">
       <widget class="QComboBox" name="mixModeComboBox">
        <property name="toolTip">
         <string>The method used to mix this Layer with its base Section</string>
        </property>
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Alpha</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Multiply</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Overlay</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="activeLayerComboBox">
        <property name="toolTip">
         <string>The currently selected Layer</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="activeLayerLabel">
        <property name="text">
         <string>Active Layer</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="mixModeLabel">
        <property name="text">
         <string>Mix Mode</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="alphaLabel">
        <property name="text">
         <string>Alpha</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="alphaSpinBox">
        <property name="toolTip">
         <string>The Layer's transparency</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="gridSettingsGroupBox">
     <property name="title">
      <string>Grid Settings</string>
     </property>
     <layout class="QGridLayout" name="gridSettingsLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="scrollLabel">
        <property name="text">
         <string>Scroll</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="scrollXSpinBox">
        <property name="toolTip">
         <string>Sets the scroll rate/direction along the x-axis</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="minimum">
         <number>-60000</number>
        </property>
        <property name="maximum">
         <number>60000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="scrollYSpinBox">
        <property name="toolTip">
         <string>Sets the scroll rate/direction along the y-axis</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="minimum">
         <number>-60000</number>
        </property>
        <property name="maximum">
         <number>60000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="offsetLabel">
        <property name="text">
         <string>Offset</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="offsetXSpinBox">
        <property name="toolTip">
         <string>Sets the offset along the x axis</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="offsetYSpinBox">
        <property name="toolTip">
         <string>Sets the offset along the y axis</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="gridSizeLabel">
        <property name="text">
         <string>Grid Size</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="gridSizeXSpinBox">
        <property name="toolTip">
         <string>Sets the width of the Pixel grid (select Base Section to edit)</string>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="gridSizeYSpinBox">
        <property name="toolTip">
         <string>Sets the height of the Pixel grid (select Base Section to edit)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
